src/pages/guides/platform-guide/audiences/index.md

```diff
@@ -445,6 +445,8 @@ There is no limit to the number of tags you can create, but each tag name is lim
 
 If you clone an audience, it's tags will be cloned, also.
 
+![AudienceTagOriginal.gif](https://s6.gifyu.com/images/AudienceTagOriginal.gif)
```

Comment:
this should be hosted locally on the doc site - you can add it to the images folder (see other image examples above). 

Also please make sure you run it through an image optimizer such as https://imageoptim.com/mac